### PR TITLE
Add information about transport to logs

### PR DESCRIPTION
**Pull request by VadymLuchko**
In the PR, information about Android Transport is provided to the respective classes

### CODE REVIEW

File: src/components/transport_manager/src/android/android_device_scanner.cc [hunk 1]
```diff
@@ -55,13 +55,15 @@ namespace transport_adapter {
 SDL_CREATE_LOG_VARIABLE("TransportManager")
 
 AndroidDeviceScanner::AndroidDeviceScanner(
-    TransportAdapterController* controller)
+    AndroidTransportAdapter* controller)
     : controller_(controller)
     , ipc_control_receiver_(new LocalSocketReceiver(
      std::bind(&AndroidDeviceScanner::ProcessMessage, this, std::placeholders::_1))) {
 
+  ipc_control_receiver_->SetTransportName(controller->GetTransportName());
```
Comment (AKalinich-Luxoft) [inline src/components/transport_manager/src/android/android_device_scanner.cc:63]: @VadymLuchko I would rename this to `SetChannelName`

File: src/components/transport_manager/src/android/local_socket_sender.cc [hunk 2]
```diff
@@ -24,6 +24,7 @@ LocalSocketSender::LocalSocketSender(OnDataSentCallback&& sent_callback, OnConne
     , message_queue_()
     , connected_callback_(connected_callback)
     , sent_callback_(sent_callback)
+    , transport_name_("Unassigned")
```
Comment (AKalinich-Luxoft) [inline src/components/transport_manager/src/android/local_socket_sender.cc:27]: @VadymLuchko I think it's better to pass something like `LoggerDelegate&& callback` which accepts `std::string` as an input param. Then you don't need to store the copy of the transport name from another place. In the callback, you can access `controller_` and get the actual transport name

Comment (AKalinich-Luxoft) [reply src/components/transport_manager/src/android/local_socket_sender.cc:27]: @VadymLuchko please make it for both sender and receiver classes

File: src/components/transport_manager/src/android/local_socket_receiver.cc [hunk 3]
```diff
@@ -95,7 +96,7 @@ void LocalSocketReceiver::Run()
 }
 
 void LocalSocketReceiver::Stop() {
-    SDL_LOG_DEBUG("Requesting server to stop");
+    SDL_LOG_DEBUG("Requesting server to stop" << transport_name_);
```
Comment (AKalinich-Luxoft) [inline src/components/transport_manager/src/android/local_socket_receiver.cc:99]: @VadymLuchko please also add logging in `LocalSocketReceiver::Run`

File: src/components/transport_manager/src/android/android_transport_adapter.cc [hunk 4]
```diff
@@ -203,5 +182,24 @@ std::string AndroidTransportAdapter::GetControlReceiverSocketName() const{
     return "";
 }
 
+ std::string AndroidTransportAdapter::GetTransportName() const
+ {
+             switch(transport_type_) {
+            case AndroidTransportType::BLE:
+                return "Bluetooth Low Energy";
```
Comment (AKalinich-Luxoft) [inline src/components/transport_manager/src/android/android_transport_adapter.cc:189]: @VadymLuchko I would recommend using prefixes for easier filtration of log messages in the console:
```suggestion
                return "[IPC][CPP][BLE]";
```
Then, you can use "[IPC]" to filter out all IPC messages from both C++ and Java sides. Or you can use "[IPC][CPP]" to filter out all IPC messages from the C++ part only. Or finally, you can use "[IPC][CPP][BLE]" to filter out all BLE transport messages from the C++ part only.

File: src/components/transport_manager/src/android/android_transport_adapter.cc [hunk 5]
```diff
@@ -203,5 +182,24 @@ std::string AndroidTransportAdapter::GetControlReceiverSocketName() const{
...
+            break;
+
+            case AndroidTransportType::BT:
+                return "Bluetooth";
```
Comment (AKalinich-Luxoft) [inline src/components/transport_manager/src/android/android_transport_adapter.cc:193]:
```suggestion
                return "[IPC][CPP][BT]";
```

File: src/components/transport_manager/src/android/android_transport_adapter.cc [hunk 6]
```diff
@@ -203,5 +182,24 @@ std::string AndroidTransportAdapter::GetControlReceiverSocketName() const{
...
+            break;
+
+            default:
+                SDL_LOG_ERROR("Unknown android transport type");
+            break;
+        }
+
+        return "Unknown";
```
Comment (AKalinich-Luxoft) [inline src/components/transport_manager/src/android/android_transport_adapter.cc:201]:
```suggestion
        return "[IPC]["Unknown]";
```

File: src/components/transport_manager/src/android/android_transport_adapter.cc (hunk 4 above)
Comment (AKalinich-Luxoft) [reply src/components/transport_manager/src/android/android_transport_adapter.cc:189]: @VadymLuchko for log callbacks in receiver and sender classes, you can also append [SENDER] and [RECEIVER] labels correspondingly

**Review by AKalinich-Luxoft — COMMENTED** [review]
@VadymLuchko please make the same changes and the same label approach for a Java part IPC classes as well.
I would rebase this PR onto classic BT branch of @ychernysheva as the biggest difficulties with logging is there

**Comment by VadymLuchko** [comment]
@AKalinich-Luxoft  Since this PR was created before merging with the release branch, I decided to recreate it by branching from the merged android_poc branch.
New RP is https://github.com/CustomSDL/sdl_core/pull/308 where I processed all comments

**Pull request closed without merging**